src/modules/shared/pages/innovation/sections/section-info.component.ts

```diff
@@ -46,47 +32,40 @@ export class PageInnovationSectionInfoComponent extends CoreComponent implements
 
   baseUrl: string;
 
-  // Flags
-  isInnovatorType: boolean;
-  isAccessorType: boolean;
-  isAssessmentType: boolean;
-  shouldShowDocuments = false;
+  paramSubscription: Subscription = new Subscription();
+
+  sectionIdentification: {
+    group: {
+        number: number;
+        title: string;
+    };
+    section: {
+        number: number;
+        title: string;
+    }
+  } | null = null;
+
 
   constructor(
     private activatedRoute: ActivatedRoute,
-    private innovationDocumentsService: InnovationDocumentsService
   ) {
 
     super();
 
+    this.sectionId = this.activatedRoute.snapshot.params.sectionId;
     this.innovation = this.stores.context.getInnovation();
 
-    this.section = {
-      id: this.activatedRoute.snapshot.params.sectionId,
-      nextSectionId: null,
-      title: '',
-      status: { id: 'UNKNOWN', label: '' },
-      isNotStarted: false,
-      submitButton: { show: false, label: "Confirm section answers" },
-      hasEvidences: false,
-      wizard: new WizardEngineModel({}),
-      date: '',
-      submittedBy: null,
-      openTasksCount: 0
-    };
-
     this.sectionsIdsList = getInnovationRecordConfig().flatMap(sectionsGroup => sectionsGroup.sections.map(section => section.id));
 
     this.baseUrl = `${this.stores.authentication.userUrlBasePath()}/innovations/${this.innovation.id}`;
 
-    // Flags
-    this.isInnovatorType = this.stores.authentication.isInnovatorType();
-    this.isAccessorType = this.stores.authentication.isAccessorType();
-    this.isAssessmentType = this.stores.authentication.isAssessmentType();
-
   }
 
   ngOnInit(): void {
+    
+    this.paramSubscription = this.activatedRoute.params.subscribe(() => {
```

Comment:
Here you probably would use the subscription available from inside core component. You can search for something like this on the FE to see some use cases:

this.subscriptions.push(
      this.activatedRoute.params.subscribe(

Reply:
This is great! Didn't know the core component had this (probably should have, since it's used on line 74). 
Will move 'this.activatedRoute.params.subscribe' to that existing one.